### PR TITLE
updates in card component

### PR DESCRIPTION
**Pull request by shubhmehra**
Updates of card components

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  gap: 1rem;
+  flex-flow: column;
+  height: max-content;
+  max-width: 300px;
```
Comment (sumitch29) [inline components/card/card.css]: Try to use rem.

**Review by sumitch29 — APPROVED** [review]
Good to go.

**Review by harshitpaliwal95 — APPROVED** [review]
looks good

File: components/card/card.css (hunk 1 above)
Comment (shubhmehra) [reply components/card/card.css]: yup right fixing it

**Review by samyakshah3008 — APPROVED** [review]
Looks good to merge!

**Pull request merged**